application/F3DStarter.cxx

```diff
@@ -286,6 +287,13 @@ int F3DStarter::Start(int argc, char** argv)
           }
           return true;
         }
+
+        if (keySym == "F12")
```

Comment:
any specific reason to put that in the app and not in the lib ?

Reply:
100% because it was easier that way tbh :D
But it could go on the lib side instead, yes. The output filename templating could be reused by `--output` too possibly.